src/test/java/org/jenkinsci/plugins/workflow/cps/FlowDurabilityTest.java

```diff
@@ -192,7 +193,7 @@ static WorkflowRun createAndRunSleeperJob(Jenkins jenkins, String jobName, FlowD
         provider.registerHint(jobName, durabilityHint);
         job.setDefinition(def);
         WorkflowRun run = job.scheduleBuild2(0).getStartCondition().get();
-        Thread.sleep(4000L);  // Hacky but we just need to ensure this can start up
+        story.j.waitForMessage("Sleeping for", run);
```

Comment:
May make this less flaky. Noticed because it prevented me from using the debugger.